test/gui/shared/scripts/helpers/WinPipeHelper.py

```diff
@@ -0,0 +1,133 @@
+import os
+import time
+import win32pipe, win32file, winerror, pywintypes, win32event
+
+TIMEOUT = 100
+DEFAULT_BUFLEN = 4096
+CLIENT_MESSAGES = [
+    'STATUS',
```

Comment:
They are the same on all platforms maybe we can share them?

Reply:
yeah, but as of now I don't see other usages. we don't need to filter out socket messages in linux and also the socket connection helper file is from https://raw.github.com/owncloud/client-desktop-shell-integration-nautilus/master/src/syncstate.py, so I won't dare to edit it :sweat_smile: